src/tunacode/ui/renderers/tools/discover.py

```
@@ -161,7 +161,7 @@ def _find_summary(lines: list[str]) -> str:
                 return stripped
         return ""
 
-    def parse_result(self, args: dict[str, Any] | None, result: str) -> DiscoverData | None:
+    def parse_result(self, args: ToolArgs | None, result: str) -> DiscoverData | None:
```

Ruff (0.15.20)

[warning] Unused method argument: `args`

(ARG002)